### ISSUE TITLE
有預期 CVAE facescrub-5 loss 收斂在多少嗎?

### ISSUE THREAD

**Issue by chyushih** [2018-summer-DL-training-program/Lab-3-CVAE-InfoGAN#2]
Dear TA, 
1. MNIST 可以成功收斂在 loss = 5000 附近 (iter 98300 的時候還可以下降到 loss = 2712) 
2. 但我用同樣的 MNIST model  run facescrub-5 loss 卻一直在 43000 附近徘迴
想請問是否有預期 loss 下降到多少附近? 
有建議的參考文件 / link / paper ?
